### ISSUE TITLE
ng2-datepicker in reactive forms , angular 2

### ISSUE THREAD

**Issue by aruribadraiah** [valor-software/ngx-bootstrap#2054]
can anybody help in using reactive forms in anguarl 2 for the ng2-datepicker.

**Comment by KChadwick96** [comment]
+1

**Comment by Simon-Briggs** [comment]
To get the basics working all you need is the FormControlName attribute. 
```
<form [formGroup]="formGroup">
        <datepicker formControlName="date"></datepicker>
</form>
```

**Comment by preetham1290** [comment]
+1

**Comment by valorkin** [comment]
check demo for reactive forms and new datepicker
available in v1.9.2
http://valor-software.com/ngx-bootstrap/#/datepicker
